python_modules/libraries/dagster-postgres/dagster_postgres/utils.py

```diff
@@ -156,15 +156,8 @@ def pg_alembic_config(dunder_file, script_location=None):
 
 
 @contextmanager
-def create_pg_connection(engine, _alembic_config, storage_type_desc=None):
+def create_pg_connection(engine, **kwargs):  # pylint: disable=unused-argument
```

Comment:
kwargs to make internal changes not have to be lock step

Reply:
fair enough although keeping them in lockstep isn't that hard?